networkx/algorithms/approximation/connectivity.py

```diff
@@ -244,6 +244,18 @@ def all_pairs_node_connectivity(G, nbunch=None, cutoff=None):
     K : dictionary
         Dictionary, keyed by source and target, of pairwise node connectivity
 
+    Examples
+    --------
+    >>> # A 3 node cycle with one extra node attached has connectivity 2 between all
+    >>> # nodes in the cycle and connectivity 1 between the extra node and the rest
```

Comment:
Just a quick note: you can actually include text normally in the `Examples` section, as long as there is a blank line between "normal" text and the executable examples:

```suggestion
    A 3 node cycle with one extra node attached has connectivity 2 between all
    nodes in the cycle and connectivity 1 between the extra node and the rest:
    
```

Reply:
Great!